lib/build.js

```diff
@@ -21,6 +21,7 @@ exports.usage = 'Invokes `' + (win ? 'msbuild' : 'make') + '` and builds the mod
 function build (gyp, argv, callback) {
   var release = processRelease(argv, gyp, process.version, process.release)
     , makeCommand = gyp.opts.make || process.env.MAKE
+      || (process.platform.indexOf('aix') != -1 && 'gmake')
```

Comment:
it is kind of of ugly.  What do you think of this. It's more verbose but also more obvious whats going on 

function build (gyp, argv, callback) {
  var platformMake = 'make';
  if (process.platform == 'aix') {
    platformMake = 'gmake';
  } else if (process.platform.indexOf('bsd') != -1) {
    platformMake = 'gmake');
  }

  var release = processRelease(argv, gyp, process.version, process.release)
    , makeCommand = gyp.opts.make || process.env.MAKE || platformMake
    , command = win ? 'msbuild' : makeCommand
    , buildDir = path.resolve('build')
    , configPath = path.resolve(buildDir, 'config.gypi')
    , jobs = gyp.opts.jobs || process.env.JOBS
    , buildType
    , config
    , arch
    , nodeDir
    , copyDevLib